modules/RTC/RTCUtils.js

```diff
@@ -847,6 +847,15 @@ var RTCUtils = {
 
                 //AdapterJS.WebRTCPlugin.setLogLevel(
                 //    AdapterJS.WebRTCPlugin.PLUGIN_LOG_LEVELS.VERBOSE);
+
+                AdapterJS.WebRTCPlugin.isPluginInstalled(
```

Comment:
Yes, the Temasys UI is shown before the initialize flow is invoked. The only change in behavior is that it now rejects the promise returned from the initialize flow instead of leaving it open forever.